Examples/ObjectiveC/StudioClassicStyleExample.m

```diff
@@ -1,9 +1,9 @@
-#import "CustomRasterStyleExample.h"
+#import "StudioClassicStyleExample.h"
```

Comment:
This isn’t entirely accurate — it can be used with any XYZ rasters, not just ones created with Studio Classic or hosted with Mapbox. The note [on the examples page](https://www.mapbox.com/ios-sdk/examples/raster-styles/) makes that clear.

Reply:
If I’m not mistaken, the example’s title would mention Studio Classic to avoid the (over)use of the word “custom”. Perhaps “raster style” is good enough?

Reply:
Yeah, this was changed to reflect the upcoming change we're making on the live examples page, since it will soon be `Apply a style designed in Mapbox Studio Classic` instead of `Custom raster style` - I would rather have the titles match similarly between the examples page and the ones that live in here.